flight/Modules/LQGSolver/lqgsolver.c

```diff
@@ -0,0 +1,117 @@
+#include "pios.h"
+#if defined(PIOS_INCLUDE_LQG)
+
+#include "pios_thread.h"
+#include "pios_modules.h"
+#include "taskmonitor.h"
+#include "taskinfo.h"
+
+#include "lqg.h"
+
+#include "lqgsolution.h"
+
+#define STACK_SIZE_BYTES 600		// Reevaluate.
+#define TASK_PRIORITY				PIOS_THREAD_PRIO_LOW
+
+/* Hack things into pieces to give other threads more time. Seems like USB telemetry goes
```

Comment:
Can we do this in stabilization instead of a dedicated task?  Saves RAM, etc.  Just do a small number of iterations each stabilization cycle.  Also I don't really like more modules.